apps/champions/lib/champions/users.ex

```diff
@@ -88,19 +83,37 @@ defmodule Champions.Users do
     Currencies.add_currency(user.id, Currencies.get_currency_by_name!("Gems").id, 500)
   end
 
+  defp add_campaigns_progress(user) do
+    campaigns = GameBackend.Campaigns.get_campaigns()
+
+    Enum.each(campaigns, fn campaign ->
+      if campaign.campaign_number == 1,
```

Comment:
What is this `campaign_number == 1` for?

Reply:
It's the first campaign of the SuperCampaign!